src/index.js

```diff
@@ -0,0 +1,69 @@
+
+const path = require('path');
+const fs = require('fs');
+const fetch = require('node-fetch');
+
+
+
+const inquirer = require('inquirer');
+inquirer
+  .prompt({
+   name: 'archivo',
+   message: '¿Que archivo deseas leer',
+       
+  })
+  .then(answers => {
+      let archivo = answers.archivo;
+     console.log(archivo)
+     validateMd(archivo)
+  })
```

Comment:
Me gustaria ver una mejor definición
```suggestion
  .then(result => {
      let file = result.file;
 
     validateMd(files)
  })
```